doc/user/ripngd.rst

```diff
@@ -22,62 +22,46 @@ ripngd Configuration
 
 Currently ripngd supports the following commands:
 
-.. index:: router ripng
 .. clicmd:: router ripng
 
    Enable RIPng.
 
-.. index:: flush_timer TIME
 .. clicmd:: flush_timer TIME
 
    Set flush timer.
 
-.. index:: network NETWORK
 .. clicmd:: network NETWORK
 
    Set RIPng enabled interface by NETWORK.
 
-.. index:: network IFNAME
 .. clicmd:: network IFNAME
 
    Set RIPng enabled interface by IFNAME.
 
-.. index:: route NETWORK
 .. clicmd:: route NETWORK
 
    Set RIPng static routing announcement of NETWORK.
 
-.. index:: router zebra
-.. clicmd:: router zebra
```

Comment:
does the description of this mean that there _is_ a "no" form that's meaningful?

Reply:
command doesn't exist at all